app/experimenter/nimbus-ui/src/pages/ExperimentNew/index.test.tsx

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
When working on `fxa-settings`, I had proposed to the team we place our "pages" in `components` since `pages` are still React components, and ended up creating `components/_pages/` so it'd be at the top of the tree, which I didn't end up liking. 

[See this thread ](https://mozilla.slack.com/archives/G0177C5FTCZ/p1598983640019000?thread_ts=1598979686.013700&cid=G0177C5FTCZ) where we talk about a `pages/` dir too - we ended up agreeing on a flat directory structure inside components, in which case this would be named `PageExperimentNew`.

We also talked about a flat directory structure [in the FE ADR](https://github.com/mozilla/experimenter/pull/3392#discussion_r485067560) where I reference `fxa-settings`.

_However_, I have fiddled a bit with NextJS on the side since these discussions and the convention there is to use `pages/` like you are here. Do you have a strong preference for `pages/ExperimentNew/` rather than `PageExperimentNew`? If so I'm fine with it and betting the rest of the team is too but we might want to double check with Jody and Jared just in case they strongly prefer the flat structure. 🙂

Reply:
> we place our "pages" in components since pages are still React components

Personally, I'd rather have `pages/` separate from `components/` - while pages *are* technically components, they're different enough in purpose & usage & structure to be distinct, IMO.

Maybe arbitrary, but I think "components" should be mostly for presentation whereas pages / routes should handle composition & orchestration of other resources that feed into presentation components. That's how we organized things on payments-server and kind of to match content-server.

Reply:
I prefer flat since I can never guess which folder a component I need is in, but that's just my vote 🖐️

Reply:
While I'm personally more partial to a flat structure, I'm happy to fall back here if the consensus is having distinct `pages` components 😄 I guess I'd want to know what belongs in a page component and how strict we should be -- should _anything_ that's not specific to layout be allowed to live in a page component? My guess is no.

Reply:
Maybe it's overkill, but this kinda feels worth a super-tiny ADR?

Reply:
> Maybe it's overkill, but this kinda feels worth a super-tiny ADR?

Or just a quick edit of this line if we go the `pages/` route:
https://github.com/mozilla/experimenter/pull/3392/files#diff-3a526648f7d71c5ec63e63771f901fd86a3b6735a5cb16928900b19469ab902cR50

I made an after-merged edit here: https://github.com/mozilla/experimenter/pull/3486/files

Reply:
I mostly missed the original FE ADR discussion, so I didn't catch the mentions of a single component directory. I can just move things in this PR. Not my preference - but ADRs exist for a reason that supersedes preference, so I don't want to randomly bikeshed it in a PR.

> Or just a quick edit of this line if we go the pages/ route:

I wouldn't make an edit to an already-merged ADR unless it was to clarify the original decision. This sounds like it would be a change in the decision, which should be a new ADR describing if/why we changed our minds.

Reply:
> I guess I'd want to know what belongs in a page component and how strict we should be -- should anything that's not specific to layout be allowed to live in a page component?

Just to clarify this bit - it's not so much about layout. 

[A page / route / container component](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) is where complex things happen. Maybe where Redux comes in or API calls, along with process orchestration. Business & interaction logic. Tests & stories are more complex and require more mocking because these are where the meat of the app lives.

Other components should largely just be "functions" - i.e. data goes in, DOM comes out, all logic is presentational. Tests & stories are simple because these are mainly just formatting components.

Having said all the above, though, even the author of that post I linked says this isn't a pattern to be dogmatically followed. So, 🤷‍♂️